terraform/pipeline/workspace.py

```diff
@@ -5,29 +5,32 @@
 import sys
 
 REASON = os.environ["REASON"]
+
 # the name of the variable that Azure Pipelines uses for the source branch
 # depends on the type of run, so need to check both
 SOURCE = os.environ.get("OTHER_SOURCE") or os.environ["INDIVIDUAL_SOURCE"]
+
 TARGET = os.environ["TARGET"]
 
-# the branches that correspond to environments
-ENV_BRANCHES = ["dev", "test", "prod"]
+# branch to workspace/environment mapping
+ENV_BRANCHES = {"main": "dev", "test": "test", "prod": "prod"}
 
 if REASON == "PullRequest" and TARGET in ENV_BRANCHES:
     # it's a pull request against one of the environment branches, so use the
     # target branch
-    environment = TARGET
+    workspace = ENV_BRANCHES[TARGET]
 elif REASON in ["IndividualCI", "Manual"] and SOURCE in ENV_BRANCHES:
     # it's being run on one of the environment branches, so use that
-    environment = SOURCE
+    workspace = ENV_BRANCHES[SOURCE]
 else:
     # default to running against dev
-    environment = "dev"
+    workspace = "dev"
 
 # matching logic in ../init.sh
-workspace = "default" if environment == "prod" else environment
+# workspace to service connection mapping
+service_connections = {"dev": "Development", "test": "Development", "prod": "Production"}
```

Comment:
I suggest replacing `prod` with `default` here (assuming the `ENV_BRANCHES` dict is updated to map to the `prod` branch to the `default` workspace)

Reply:
Ah that's true, thank you!